amr-wind/fvm/nonLinearSum.H

```diff
@@ -0,0 +1,166 @@
+#ifndef NONLINEARSUM_H
+#define NONLINEARSUM_H
+
+#include "amr-wind/fvm/fvm_utils.H"
+
+namespace amr_wind::fvm {
+
+/** Strain rate operator
```

Comment:
This comment and a bunch of other comments for functions in this file seem to be referencing fvm::StrainRate, can you update to the non linear sum operator?